README.md

```diff
@@ -23,25 +44,83 @@ The #1 best way to understand and address the weaknesses in an API's design and
+* An overview of the contents of the API and the data sources.
+* Production URLs for accessing the API.
+* Required parameters and defaults.
+* A description of the data that is returned.
+* A description of the error codes that are returned, and their meaning.
+
+Additional nice-to-haves include:
+* Explanation of key management and a sample key.
+* Description of update frequency.
+* Interactive documentation to demonstrate sample calls.
+* Sample client code for consuming the API in common languages.
+
 ### Point of contact
 
 Have an obvious mechanism for clients to report issues and ask questions about the API.
 
-When using GitHub for an API's code, use the associated issue tracker. In addition, publish an email address for direct, non-public inquiries.
+When using GitHub for an API's code or documentation, use the associated issue tracker. In addition, publish an email address for direct, non-public inquiries.
+
+If you don't have a support channel specific to your API, you can use the issue tracker at [GSA-APIs](https://github.com/GSA/GSA-APIs/issues). Be sure your support team watches for issues there.
+
+### Avoid an API "Ghost Town": responding to issues and questions
+Developers can immediately sniff out a stale and unsupported API by old issues never answered and contact emails that no longer work.
+
+It is critical to respond to issues posted or queries submitted by developers. This demonstrates that the API can be counted on for production usage. If an immediate fix (or even a developer to investigate) is not readily available, respond anyway. Developers will be glad to know when you'll be able to take a look.
 
 ### Notifications of updates
 
 Have a simple mechanism for clients to follow changes to the API.
 
 Common ways to do this include a mailing list, or a [dedicated developer blog](https://developer.github.com/changes/) with an RSS feed.
 
-### API Endpoints
+### Decommission Unsupported APIs
+
+If an API can no longer be supported, consider decommissioning the API and removing the documentation. If the API will remain available for historical purposes without support, update the documentation to reflect this.
+
+### Avoid Breaking Changes
 
+Any changes made to a specific version of your API should not break your contract with existing users. If you need to make a change that will break that contract, create a new version at a separate URL path. Leave at least one previous version intact. And communicate to existing users to understand when previous versions will be decommissioned.
+
+## Design Considerations
+
+### API Endpoints
 An "endpoint" is a combination of two things:
 
 * The verb (e.g. `GET` or `POST`)
 * The URL path (e.g. `/articles`)
 
+The URL path should follow this pattern if possible for a collection of items:
+(path)/{business_function}/{application_name}/{version}/{plural_noun}
+
+An example would be:
+https://api.data.gov/financial_management/sample_app/v1/vendors
```

Comment:
From Cindy S: Is this link supposed to go to a specific page, it returns an error in data.gov.  If this is an example should you remove the hyperlink?